Reorg part1 redirect tutorials (#793)
solid/solidproject.org: _posts/developers/apps/inrupt-tutorial/2020-11-06-00_first-app.md

[JavaScript Client Libraries](https://github.com/inrupt/solid-client-js) to read
and write your name to your user profile.

> Note: Although this tutorial uses Inrupt's JavaScript libraries, you can use other libraries (be it for JavaScript or other languages) that support Solid. For more information, see the [Solid Developer Tools & Libraries](https://solidproject.org/developers/tools/).

Reviewer: ```suggestion
Note: Although this tutorial uses Inrupt's JavaScript libraries, you can use other libraries (whether for JavaScript or other languages) that support Solid. For more information, see the [Solid Developer Tools & Libraries](https://solidproject.org/developers/tools/).
```

Author: Hey @TallTed  - thanks for this suggestion!  
Just as an FYI ... in this pull, this content will not show.  Instead, if people had an old link to this page, this will redirect to the inrupt's getting started tutorial (which the old page used to be based on). For a preview, see this link
https://deploy-preview-793--solidproject-dot-org.netlify.app/developers/tutorials/first-app

Author: Oh ... and I'm definitely open to committing the your suggestion because it is a good change.  But, since the content doesn't show ... not sure what the group prefers.

Reviewer: I'm fairly confident that good changes are good changes, whether or not they're visible in a default view.

Also, as noted in https://github.com/solid/solidproject.org/pull/793#pullrequestreview-1536670732, the detail about Inrupt libraries *should* be visible.

   `regenerator-runtime/runtime` in your JavaScript code, which will inject a
   suitable compatibility layer.

### 4. Create the Application

In the `my-demo-app` directory, create the files for the application.

1. Create a `my-demo.css` file with the following content:

   ```css

   {% include_relative codesandbox/my-demo.css %}

   ```

1. Create an `index.html` file with the following content:

   ```html
   {% include_relative codesandbox/index.html %}
   ```

1. Create an `index.js` file with the following content:

   > **NOTE:** If your Pod is not on `https://solidcommunity.net`, modify the
   > value of the `SOLID_IDENTITY_PROVIDER` variable accordingly.

   ```javascript

   {% include_relative codesandbox/index.js %}

   ```

## Run the Application

1. In the `my-demo-app` directory, run:

   ```shell
   npx parcel index.html
   ```

   The output should resemble the following:

   ```shell
     Server running at http://localhost:1234
     ...
     Built in 2ms.
   ```

1. Open `localhost:1234` in a browser.

1. Click the `Login` button.

   - The first time you log into your Pod with this application (it'll
     be named `http://localhost:1234`), you'll

>      be prompted to authorize it to access your Pod . To allow the application to

Reviewer: ```suggestion
     be prompted to authorize it to access your Pod. To allow the application to
```

Author: Ditto.  Great catch.  But, same as before, since the content doesn't show ... not sure what the group prefers.
